dist/api_data.py

```diff
@@ -44,6 +49,234 @@ def __gt__(self, other):
+           ]),
+    ConnectionControlPoint('Thread 1', 'Wait for trigger', 'Always', '', r'''
+           Thread 2 waits for thread 1 to get here.''',
+           type='category', subconfig= [
+               # Action configuration parameters
+               Config('wait_count', '1', r'''
+                      the number of triggers for which to wait''',
+                      min='1', max='4294967295'),
+           ]),
+    ConnectionControlPoint('Thread 2', 'Wait for trigger', 'Always', '', r'''
+           Thread 3 waits for thread 2 to get here.''',
+           type='category', subconfig= [
+               # Action configuration parameters
+               Config('wait_count', '1', r'''
+                      the number of triggers for which to wait''',
+                      min='1', max='4294967295'),
+           ]),
+    ConnectionControlPoint('Thread 3', 'Wait for trigger', 'Always', '', r'''
+           Thread 4 waits for thread 3 to get here.''',
+           type='category', subconfig= [
+               # Action configuration parameters
+               Config('wait_count', '1', r'''
+                      the number of triggers for which to wait''',
+                      min='1', max='4294967295'),
+           ]),
+    ConnectionControlPoint('Thread 4', 'Wait for trigger', 'Always', '', r'''
+           Thread 5 waits for thread 4 to get here.''',
+           type='category', subconfig= [
+               # Action configuration parameters
+               Config('wait_count', '1', r'''
+                      the number of triggers for which to wait''',
+                      min='1', max='4294967295'),
+           ]),
+    ConnectionControlPoint('Thread 5', 'Wait for trigger', 'Always', '', r'''
+           Thread 6 waits for thread 5 to get here.''',
+           type='category', subconfig= [
+               # Action configuration parameters
+               Config('wait_count', '1', r'''
+                      the number of triggers for which to wait''',
+                      min='1', max='4294967295'),
+           ]),
+    ConnectionControlPoint('Thread 6', 'Wait for trigger', 'Always', '', r'''
+           Thread 7 waits for thread 6 to get here.''',
+           type='category', subconfig= [
+               # Action configuration parameters
+               Config('wait_count', '1', r'''
+                      the number of triggers for which to wait''',
+                      min='1', max='4294967295'),
+           ]),
+    ConnectionControlPoint('Thread 7', 'Wait for trigger', 'Always', '', r'''
+           Thread 8 waits for thread 7 to get here.''',
+           type='category', subconfig= [
+               # Action configuration parameters
+               Config('wait_count', '1', r'''
+                      the number of triggers for which to wait''',
+                      min='1', max='4294967295'),
+           ]),
+    ConnectionControlPoint('Thread 8', 'Wait for trigger', 'Always', '', r'''
```

Comment:
Is there a reason we chose to have 9 threads here? I think the example could be simpler with 3 control points?

Reply:
* To have authenticity. The more threads, the less likely it would get the correct order by chance.
* Because there are more than one print thread, this demonstrates that multiple control points can be defined by the same DEFINE control point macro call by just varying the ID.

Done. I will reduce it to 5 print threads.